tests/UKHO.S100PermitService.Common.UnitTests/Encryption/S100CryptTests.cs

```diff
@@ -101,5 +111,92 @@ private List<ProductKeyServiceResponse> GetInvalidProductKeyServiceResponse()
                 }
             ];
         }
+
+        [Test]
+        public void WhenValidMKeyAndUpnInfo_ThenListOfDecryptedHardwareIdIsReturned()
+        {
+            const string FakeDecryptedHardwareId = "86C520323CEA3056B5ED7000F98814CB";
+
```

Comment:
Remove space in two constant declaration

Reply:
Done